pkg/lib/cockpit-components-onoff.jsx

```diff
@@ -29,8 +30,8 @@ import "./cockpit-components-onoff.scss";
  *            when embedding this into a non-React page)
  * text       optional string that appears to the right of the button
  */
-export const OnOffSwitch = ({ state, onChange, text, disabled, id }) => (
-    <label id={id} className="onoff-ct">
+export const OnOffSwitch = ({ state, onChange, text, disabled, id, className }) => (
+    <label id={id} className={cockpit.format("onoff-ct $0", className || "")}>
```

Comment:
cockpit.format() is too heavy for this -- `"onoff-ct " + (className || "")`  will do nicely.

Reply:
Dropped this commit - it was not needed any more.